test/test_files/comments.dart

```diff
@@ -11,6 +11,12 @@
 /// ...
 var a;
 
+/// ``
+var noInlineCode;
+
+/// ` `
+var inlineCode;
+
```

Comment:
Apparently we don't ahve anye xisting tests for this with non-whitespace contents - perhaps it's worth adding one (because it's not obvious from this test whether whitespace might be special rather than just treated the same as other content).

Reply:
Good point! I added an inline code span golden with more than just whitespace :)